pkg/controllers/user/rbac/roletemplate_handler.go

```diff
@@ -46,7 +53,7 @@ func (c *rtLifecycle) Updated(obj *v3.RoleTemplate) (runtime.Object, error) {
 		return nil, nil
 	}
 
-	err = c.syncRT(obj, hasPRTBs)
+	err = c.syncRT(obj, hasPRTBs, prtbs)
```

Comment:
What is there are no prtbs, but there are crtbs present? can that be the case? hasPRTBs will be true and prtbs[] will be empty in that case

Reply:
I guess we need to also pass the crtbs to syncRT?

Reply:
The prtb_handler sync wasn't getting called because the change was on roletemplate. But I have updated the pr with logic to pass crtbs too so the clusterrolebindings created for crtbs are updated